### ISSUE TITLE
Rotation option is not working

### ISSUE THREAD

**Issue by ghost** [skokec/segdec-net-jim2019#24]
Hi,

I just tried to use the rotation=True option in the code however it seems not working: 

File "../segdec-net-jim2019/segdec_model.py", line 258, in get_loss
    l1 = tf.losses.sigmoid_cross_entropy(logits=predictions,multi_class_labels=masks)
InvalidArgumentError (see above for traceback): Incompatible shapes: [1,32,88,1] vs. [1,88,32,1]

I tried to solve it myself however failed. May I know how I can fix this problem?

Cheers
